docs/recipes/es-modules-with-@std-esm.md

```diff
@@ -0,0 +1,109 @@
+# ES Modules
+
+As of Node.js 8.5.0, [ES modules](http://2ality.com/2017/09/native-esm-node.html) are natively supported, but behind a command line option `--experimental-modules`.
+
+However, it works a bit different on Node.js than in web browsers that also natively supports ES modules. In Node.js, you need to use the `.mjs` extension instead of the commonly known `.js` extension. The `.mjs` extension in web browsers too, but they have to be served with the correct media type (`application/javascript`). There is a [ongoing effort](https://tools.ietf.org/html/draft-bfarias-javascript-mjs-00) on standardizing `.mjs`.
+
+The good news is that there's a module called [`@std/esm`](https://github.com/standard-things/esm) that enables you to write and run ES modules in Node.js 4 and above.
+
+Here's how you get it working with AVA.
+
+
+## @std/esm
+
+First, install [`@std/esm`](https://github.com/standard-things/esm):
+
+```
+$ npm install @std/esm
+```
+
+Modify your package.json accordingly:
+
+```json
+{
+	…
+	"scripts": {
+		"test": "ava"
+	},
+	"dependencies": {
+		"@std/esm": "^0.16.0"
```

Comment:
Latest is `0.17.0` now. Is there a standard way to express a dependency without using a version for the recipe so folks don't lazy copy pasta an out of date version?

Reply:
I don't know… perhaps the recipe should say to use `npm install` and not show any `package.json`? That would make copying really hard.